test/cmdlineTests/standard_optimizer_yulDetails_optimiserSteps_no_yul_blank_spaces/input.json

```
@@ -0,0 +1,20 @@
+{
+    "language": "Solidity",
+    "sources": {
+        "A": { "content": "
+                    // SPDX-License-Identifier: GPL-3.0
+                    pragma solidity >=0.0;
+                "
+            }
+    },
+    "settings": {
+        "optimizer": {
+            "details": {
+                "yul": false,
+                    "yulDetails": {
+                        "optimizerSteps": " "
+                    }
```

This is formatted in a misleading way. There's no nesting here, but at a glance it looks as if there was.
```suggestion
                "yul": false,
                "yulDetails": {
                    "optimizerSteps": " "
                }
```

---

Fixed.